src/cloud_provider/aws/databases/mongodb.rs

```diff
@@ -391,21 +390,21 @@ impl Create for MongoDB {
                         &context,
                     ),
                 )?;
-
+                // overwrite with our chart values
                 let chart_values = format!(
                     "{}/common/chart_values/mongodb",
                     &self.context.lib_root_dir()
                 );
 
-                let copy_res = crate::fs::copy_files(
-                    Path::new(chart_values.as_str()),
-                    Path::new(workspace_dir.as_str()),
-                    false,
-                );
-                match copy_res {
-                    Ok(o) => info!("Chart values are successfully copied"),
-                    _ => error!("Chart values are not copied"),
-                };
+                let _ = from_simple_error_to_engine_error(
```

Comment:
Is it possible to rename this to a more comprehensive stuff. Or at least comments